persistence/sql/migrations/legacy/20220415152108_credentials_types_saml.up.fizz

```diff
@@ -0,0 +1 @@
+sql("INSERT INTO identity_credential_types (id, name) SELECT 'ff5a1823-8b47-4255-860f-4b70ed122740', 'saml' WHERE NOT EXISTS ( SELECT * FROM identity_credential_types WHERE name = 'saml')")
```

Comment:
Sorry for missing context: what is behind `ff5a1823-8b47-4255-860f-4b70ed122740` ?